src/main/java/uk/gov/hmcts/darts/task/runner/impl/UnstructuredToArmAutomatedTask.java

```diff
@@ -36,9 +36,9 @@ protected void runTask() {
     @Override
     protected void handleException(Exception exception) {
         if (exception instanceof NullPointerException npe) {
```

Comment:
not sure why we catch a nullpointer exception seems weird. Just log the exception

Reply:
There are so many points of failure in this automated task so if there is a null pointer only we want to log the stack trace. All other exceptions we don't want the stack trace.